### PR TITLE
Fixes #25715 - Update ISS with perm and SELinux info.

### PR DESCRIPTION
**Pull request by chris1984**
Updated to use warn/info blocks as we do with Katello install.md so it looks much nicer.

### CODE REVIEW

File: plugins/katello/nightly/user_guide/content_view_import_export/index.md [hunk 1]
```diff
@@ -6,19 +6,19 @@ version: nightly
 
 # Content View Import/Export
 
-Content view import/export is one of the new features in Katello 3.9. This feature is for users who want the *exact* same content view version on one Katello instance to be available on another Katello instance. The other Katello instances may or may not have Internet access.
+* Content view import/export is one of the new features in Katello 3.9. This feature is for users who want the *exact* same content view version on one Katello instance to be available on another Katello instance. The other Katello instances may or may not have Internet access.
```
Comment (ehelms) [inline plugins/katello/nightly/user_guide/content_view_import_export/index.md]: Was wondering why the shift from paragraphs to bullet points?

Comment (chris1984) [reply plugins/katello/nightly/user_guide/content_view_import_export/index.md]: @ehelms I was trying to make it look a bit more formatted and nicer. I am fine with keeping it like the rest of the docs, I was just trying something new wrt formating.

File: plugins/katello/nightly/user_guide/content_view_import_export/index.md [hunk 2]
```diff
@@ -100,38 +100,42 @@ The `hammer content-view version export` command gathers information about a con
...
 
-Please use [foreman-ansible-modules](https://github.com/theforeman/foreman-ansible-modules) or Hammer scripts to define your SOE in a reproducible way. Once you have a reproducible SOE, you can then use import/export to keep your Katello updated.
+* Please use [foreman-ansible-modules](https://github.com/theforeman/foreman-ansible-modules) or Hammer scripts to define your SOE in a reproducible way. Once you have a reproducible SOE, you can then use import/export to keep your Katello updated.
 
+* To ensure proper SELinux contexts on the exporting tar and files, use the `/var/lib/pulp/katello-export` export directory. This directory already has the correct permissions and correct SELinux labels, and was created specifically as a landing place for files not created by Pulp that Pulp needs to read or write.
```
Comment (beav) [inline plugins/katello/nightly/user_guide/content_view_import_export/index.md]: `/var/lib/pulp/katello-export` is needed more when importing than when exporting. The user is able to export anywhere that the user running hammer can write to, but the tar file(s) need to be in `/var/lib/pulp/katello-export` when importing.

I would also mention again that the alternative is to set file permissions and selinux labels if the user chooses to use a different directory.

Comment (chris1984) [reply plugins/katello/nightly/user_guide/content_view_import_export/index.md]: @beav updated

File: plugins/katello/nightly/user_guide/content_view_import_export/index.md (hunk 1 above)
Comment (ehelms) [reply plugins/katello/nightly/user_guide/content_view_import_export/index.md]: I get your thinking. For me, it starts to feel like bullet point spam as these are not lists of something or if they are they lack some intro context. For example, if oyu added a sentence or two that led into what import/export was and that it has changed from previous versions in the following ways followed by this list that would gel better.

File: plugins/katello/nightly/user_guide/content_view_import_export/index.md [hunk 3]
```diff
@@ -100,38 +100,42 @@ The `hammer content-view version export` command gathers information about a con
...
+### Hammer import
 
-Before you run the `import` command for the first time, you will need to create the same products on the importing Katello that you had on the exporting Katello. This is a step you will only need to do once. You will also need to create the same content view, with same label. Again, you will only need to do this once.
+* The `hammer content-view version import` command uses the tar file created from the `export` command to create a content view version with the same data. It will create a content view version with the same major and minor version numbers, and the same repositories with the same packages and errata.
 
-The `import` command will synchronize the packages from the export tar file into Library. It will then call the `/publish` API and create a new content view version using those packages.
+* Before you run the `import` command for the first time, you will need to create the same products on the importing Katello that you had on the exporting Katello. This is a step you will only need to do once. You will also need to create the same content view, with same label. Again, you will only need to do this once.
```
Comment (ehelms) [inline plugins/katello/nightly/user_guide/content_view_import_export/index.md]: This section as bullets doesn't follow for me. I think they worked better as paragraphs.

**Comment by chris1984** [comment]
@ehelms updated.

**Review by ehelms — APPROVED** [review]

**Pull request merged**